ci/check-documentation.py

```diff
@@ -114,8 +114,25 @@ def complain(message):
             if md_path in index_child_md_links:
                 referenced_md_files.add(target_path)
 
-        # check for existence of links to README.md files of models and demos
+        # check <omz_dir> link validity
+
+        for link in sorted([link for link in doc_page.omz_references() if link.startswith('<omz_dir>')]):
+            file_path = Path(link.replace('<omz_dir>', str(OMZ_ROOT)))
```

Comment:
This will replace all occurrences of `<omz_dir>` in the string, but only the initial occurrence should be replaced.